lib/components/src/Accordion/AccordionHeader.tsx

```diff
@@ -0,0 +1,199 @@
+import { styled } from '@storybook/theming';
+import React, { FC, useContext, useEffect, useRef, useState } from 'react';
+import { Icons } from '../icon/icon';
+import { AccordionItemContext } from './AccordionItemContext';
+
+export type AccordionHeaderProps = {
+  label?: string;
```

Comment:
This should just always be `children` I think. Remove the `label` prop. I don't see it being useful, it's just more code-paths to support.

Reply:
Then I can not provide as good accessibility.

Label with a type and a word to describe what it does is IMO a lot more useful than just piping in children with no control - and by now almost an industry standard when making components. (I see `title` used a lot of places - which is also a mistake)

This way you can either decide to control it yourself (and take over accessibility responsibilities) - or let us do it.

This is a generic type component, and I have tried to provide to paths - the controlled and the uncontrolled.